### PR TITLE
Adds VPN Moment page 16

### PR DESCRIPTION
**Pull request by nathan-barrett**
## One-line summary
Adds VPN Moment page 16

## Significant changes and points to review
Matches with welcome page 14, so I reused a majority of the code that was used there - but updated the svg's to be purple instead of green.

## Testing
http://localhost:8000/en-US/firefox/welcome/16
https://www-demo7.allizom.org/en-US/firefox/welcome/16/

### CODE REVIEW

**Comment by nathan-barrett** [comment]
waiting on approval from stakeholders, so this can be reviewed but shouldn't be merged yet.

File: bedrock/firefox/templates/firefox/welcome/page16.html
Comment (reemhamz) [inline bedrock/firefox/templates/firefox/welcome/page16.html:62]: **Suggestion (design)**: For this section, since the text for both pictos is very short, it would look neater with centered text, since it won't compromise on readability very much:

<kbd>Screenshot 2023-10-12 at 11 00 48 AM</kbd>

**Review by reemhamz — CHANGES_REQUESTED** [review]
There are just a few things to cover, with one being a blocking request.

Great work on this so far, though! It looks super neat and clean 🧽

File: bedrock/firefox/templates/firefox/welcome/page16.html
Comment (nathan-barrett) [reply bedrock/firefox/templates/firefox/welcome/page16.html:37]: I'll double check with them, but I think that's how they want it?

Comment (reemhamz) [reply bedrock/firefox/templates/firefox/welcome/page16.html:37]: It doesn't gramatically make sense without the `a`

**Comment by reemhamz** [comment]
Omg, I'm so sorry I merged it without thinking even though it's a do-not-merge

**Pull request merged**
